webflow-hacks/misc-hacks/12/12.ts

```diff
@@ -0,0 +1,17 @@
+// when DOM ready
+document.addEventListener('DOMContentLoaded', function () {
+  // selector constants
+  const BUTTON_SELECTOR = '[fs-hacks-element="hack-button"]';
+  const TEXT_SELECTOR = '[fs-hacks-element="hack-text"]';
+  // get button and text elements
+  const button = document.querySelector(BUTTON_SELECTOR);
+  const text = document.querySelector(TEXT_SELECTOR);
```

Comment:
I think this variable being called `text` might be confusing. Is it holding just a string? Is it holding a text element like a paragraph?
I'd call it something more self explanatory like `textElement` or similar.

Reply:
How to hack another device full control